thumbor/result_storages/file_storage.py

```diff
@@ -55,11 +57,12 @@ def get(self, callback):
             logger.debug("[RESULT_STORAGE] image not found at %s" % file_abspath)
             callback(None)
         else:
-            buffer = None
+            result = ResultStorageResult()
 
             with open(file_abspath, 'r') as f:
-                buffer = f.read()
-            callback(buffer)
+                result.buffer = f.read()
+
+            callback(result)
```

Comment:
Before returning the result the file_storage should set some metadata like what is being done in `last_updated`.